IF/src/main/java/com/github/stefvanschie/inventoryframework/pane/component/ToggleButton.java

```diff
@@ -44,11 +44,11 @@ public ToggleButton(int length, int height) {
         super(length, height);
 
         this.enabledPane = new OutlinePane(0, 0, length, height);
-        this.enabledPane.addItem(new GuiItem(new ItemStack(Material.GREEN_STAINED_GLASS_PANE)));
+        this.enabledPane.addItem(new GuiItem(Objects.requireNonNull(XMaterial.GREEN_STAINED_GLASS_PANE.parseItem())));

```

Comment:
Try to avoid `requireNonNull` method invokation where possible. It is unused and the compiler will throws exception if the object is returns null.